app/styles/ui/_side-by-side-diff.scss

```diff
@@ -355,6 +391,33 @@
         }
       }
 
+      &.has-check-all-control {
+        .hunk-expansion-handle {
+          padding-left: calc(var(--hunk-handle-width-with-check-all) / 2);
+          padding-right: calc(var(--hunk-handle-width-with-check-all) / 2);
+        }
+      }
```

Comment:
This is a little bit of a bummer because in the unified diff the `hunk-expansion-handle` is calculated by the js by the line numbers width. So, with adding this padding, everything lines up, but the button keyboard focus styles are now inset. It doesn't look back, just doesn't quite match the mouse hover state as it changes the background of the whole area.